### PR TITLE
Browser extension warning

### PR DESCRIPTION
**Pull request by efb4f5ff-1298-471a-8973-3d47447115dc**
---
Browser extension warning
---

**Description**
Added a warning about why u may not want to install this browser extension

**Additional information**
Also added it to the docs https://github.com/FreeTubeApp/FreeTube-Docs/pull/21

### CODE REVIEW

**Comment by 0x6e656b6f** [comment]
You can also change the extension recommendation from Privacy Redirect to LibRedirect since the former is not actively maintained anymore.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> You can also change the extension recommendation from Privacy Redirect to LibRedirect since the former is not actively maintained anymore.

See https://github.com/FreeTubeApp/FreeTube/issues/2089#issuecomment-1046118597

**Comment by 0x6e656b6f** [comment]
> > You can also change the extension recommendation from Privacy Redirect to LibRedirect since the former is not actively maintained anymore.
> 
> See [#2089 (comment)](https://github.com/FreeTubeApp/FreeTube/issues/2089#issuecomment-1046118597)

? Their [last commit](https://github.com/libredirect/libredirect/commit/4765cebc7dc221e4a2c461efd53292d43f81e53a) is two hour ago.

**Review by PikachuEXE — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ FreeTube is supported by the [Privacy Redirect](https://github.com/SimonBrazell/
 
 Download Privacy Redirect for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/privacy-redirect/) or [Google Chrome](https://chrome.google.com/webstore/detail/privacy-redirect/pmcmeagblkinmogikoikkdjiligflglb).
 
+Disclamer: Learn more about why a browser extension is bad for your [privacy](https://www.privacyguides.org/browsers/#extensions).
```
Comment (absidue) [inline README.md:21]:
```suggestion
Disclaimer: Learn more about why a browser extension is bad for your [privacy](https://www.privacyguides.org/browsers/#extensions).
```

**Review by absidue — CHANGES_REQUESTED** [review]

**Review by absidue — APPROVED** [review]

**Review by PikachuEXE — APPROVED** [review]
It's never a good PR without typo 😆

**Review by ChunkyProgrammer — APPROVED** [review]

**Pull request merged**